docs/daily-usage.md

```diff
@@ -376,6 +384,62 @@ services:
 
 </details>
 
+## Visual Studio Code Integration
+
+Generated projects have out-of-the-box integration with
+[Visual Studio Code](https://code.visualstudio.com/) to provide you with a better and
+easier-to-use experience through:
+
+- Suggested extensions
+- Launch configurations
+- Workspace definition
+- Integration with invoke
+- Debugger integration
+
+**Note**: As the configurations are defined per project and depending on the Odoo addons
+you are working on, you might need to update your configuration frequently with
+`invoke write-code-workspace-file` (done automatically when you invoke other tasks).
+
+### Debugging Python code
+
+In order to debug Python code, you need to:
+
+1. Install suggested extensions, as mentioned above
+1. Run `invoke develop`
+1. Go to the "Run" tab in VSCode, select "Start Odoo and debug Python", and click "Run".
+   The program will stop in any breakpoint you have and you will be able to analyze your
+   Python environment.
+
+### Debugging JS code
+
+In order to debug JavaScript code, you need to:
+
+1. Install suggested extensions, as mentioned above
+1. Run `invoke develop`
+1. Go to the "Run" tab in VSCode, select "Start Odoo and debug JS in [Firefox/Chrome]",
+   and click "Run". The program will stop in any breakpoint you have and you will be
+   able to analyze your JS environment.
+
+If you face some problems while debugging in Firefox or Chrome, see our [FAQ](faq.md) or
+the corresponding extensions documentation before opening an issue here.
+
+### Debugging Python + JS code 🤯
+
+In order to debug Python and JavaScript code at the same time, you need to:
+
+1. Install suggested extensions, as mentioned above
+1. Run the project with the provided `invoke` commands. (This will automatically
+   generate the VSCode configurations for you) **Note**: As the configurations are
+   defined per project and depending on the Odoo addons you are working on, you might
+   need to update your configuration frequently with `invoke write-code-workspace-file`
+   (done automatically when you do `invoke start`, for example).
```

Comment:
You don't really need to do this, because the pre launch task does this for you:

```suggestion
```